src/backend/marsha/core/tests/test_api_timed_text_track.py

```diff
@@ -2147,3 +2201,60 @@ def test_api_timed_text_track_initiate_upload_by_video_organization_admin(self):
                 },
             },
         )
+
+    @override_settings(SUBTITLE_SOURCE_MAX_SIZE=10)
+    def test_api_timed_text_track_initiate_upload_file_too_large(self):
+        """
+        Organization administrator token user initiates upload for a timed text track for a video.
```

Comment:
This is not an organisation administrator user. It is a LTI user with admin or instructor role.

Reply:
And the whole description is not accurate with the test